src/common/config.cc

```diff
@@ -946,33 +946,21 @@ int md_config_t::_get_val(const char *key, char **buf, int len) const
   if (!key)
     return -EINVAL;
 
-  string k(ConfFile::normalize_key_name(key));
-
-  config_value_t cval = _get_val(k.c_str());
-  if (!boost::get<invalid_config_value_t>(&cval)) {
-    ostringstream oss;
-    if (bool *flagp = boost::get<bool>(&cval)) {
-      oss << (*flagp ? "true" : "false");
-    } else if (float *fp = boost::get<float>(&cval)) {
-      oss << std::fixed << *fp ;
-    } else if (double *dp = boost::get<double>(&cval)) {
-      oss << std::fixed << *dp ;
-    } else {
-      oss << cval;
-    }
-    string str(oss.str());
-    int l = strlen(str.c_str()) + 1;
+  string val ;
+  if (!_get_val(key, &val)) {
+    int l = val.length()+1;
```

Comment:
no. @gmayyyha you can review your commit with the github web UI by yourself as well, see https://github.com/ceph/ceph/pull/15647/files